src/page/Node.tsx

```diff
@@ -348,6 +353,7 @@ class Node extends React.Component<INodeProps, INodeState> {
                             properties={this.state.properties}
                             updateProperties={properties => {
                                 this.setState({ properties: properties });
+                                this.props.tabManager.setChanged(this.props.tabId, true);
```

Comment:
Should i do deeper verification of changed data? User can put back original value.